tests/br_other/run.sh

```diff
@@ -136,22 +137,24 @@ for key in $(echo $default_pd_values | jq 'keys[]'); do
   fi
 done
 
-pd_settings=5
 
 # check is there still exists scheduler in pause.
-pause_schedulers=$(curl http://$PD_ADDR/pd/api/v1/schedulers?status="paused" | grep "scheduler" | wc -l)
-# There shouldn't be any paused schedulers since BR gracfully shutdown.
-if [ "$pause_schedulers" -ne "0" ];then
+pause_schedulers_after=$(curl http://$PD_ADDR/pd/api/v1/schedulers?status="paused" | grep "scheduler" | wc -l)
+if [ "$pause_schedulers" -ne "$pause_schedulers_after" ];then
```

Comment:
#559 got merged, we shouldn't revert its change...

```suggestion
pause_schedulers=$(curl http://$PD_ADDR/pd/api/v1/schedulers?status="paused" | grep "scheduler" | wc -l)
 # There shouldn't be any paused schedulers since BR gracfully shutdown.
 if [ "$pause_schedulers" -ne "0" ];then
```